config/typescript.js

```diff
@@ -10,7 +10,7 @@ module.exports = {
     'import/extensions': allExtensions,
     'import/external-module-folders': ['node_modules', 'node_modules/@types'],
     'import/parsers': {
-      '@typescript-eslint/parser': ['.ts', '.tsx', '.d.ts'],
+      '@typescript-eslint/parser': ['.ts', '.tsx'],
```

Comment:
Unrelated nit: `.d.ts` only matters in `import/extensions`. Here it's covered by `.ts`.